dataflow/gemma-flex-template/e2e_test.py

```diff
@@ -0,0 +1,154 @@
+
+3. Change directory to the sample location.
+    cd dataflow/gemma-flex-template
+
+4. Set the PYTHONPATH to where the conftest is located.
+    export PYTHONPATH=..
+
+OPTION A: Run tests with pytest, you can use -k to run specific tests
+    python -m venv env
+    source env/bin/activate
+    pip install -r requirements.txt -r requirements-test.txt
+    pip check
+    python -m pytest --verbose -s
+
+OPTION B: Run tests with nox
+    pip install nox
+    nox -s py-3.10
+
+NOTE: For the tests to find the conftest in the testing infrastructure,
+      add the PYTHONPATH to the "env" in your noxfile_config.py file.
+"""
+from collections.abc import Callable
+
+import conftest  # python-docs-samples/dataflow/conftest.py
+from conftest import Utils
+
+import pytest
+
+DATAFLOW_MACHINE_TYPE = "g2-standard-4"
+GEMMA_GCS = "gs://perm-dataflow-gemma-example-testdata/pytorch_model"
+NAME = "dataflow/gemma-flex-template/streaming"
+
+
+@pytest.fixture(scope="session")
+def test_name() -> str:
+    # Many fixtures expect a fixture called `test_name`, so be sure to define it!
+    return "dataflow/gemma-flex-template"
+
+
+@pytest.fixture(scope="session")
+def bucket_name(utils: Utils) -> str:
+    yield from utils.storage_bucket(NAME)
+
+
+@pytest.fixture(scope="session")
+def messages_topic(pubsub_topic: Callable[[str], str]) -> str:
+    return pubsub_topic("messages")
+
+
+@pytest.fixture(scope="session")
+def messages_subscription(pubsub_subscription: Callable[[str, str], str],
+                          messages_topic: str) -> str:
+    return pubsub_subscription("messages", messages_topic)
+
+
+@pytest.fixture(scope="session")
+def responses_topic(pubsub_topic: Callable[[str], str]) -> str:
+    return pubsub_topic("responses")
+
+
+@pytest.fixture(scope="session")
+def responses_subscription(pubsub_subscription: Callable[[str, str], str],
+                           responses_topic: str) -> str:
+    return pubsub_subscription("responses", responses_topic)
+
+
+@pytest.fixture(scope="session")
+def flex_template_image(utils: Utils) -> str:
+    conftest.run_cmd("gsutil", "cp", "-r", GEMMA_GCS, ".")
+    yield from utils.cloud_build_submit(NAME)
+
+
+@pytest.fixture(scope="session")
+def flex_template_path(utils: Utils, bucket_name: str,
+                       flex_template_image: str) -> str:
+    yield from utils.dataflow_flex_template_build(bucket_name,
+                                                  flex_template_image)
+
+
+@pytest.fixture(scope="session")
+def dataflow_job(
+        utils: Utils,
+        project: str,
+        location: str,
+        bucket_name: str,
+        flex_template_image: str,
+        flex_template_path: str,
+        messages_subscription: str,
+        responses_topic: str,
+) -> str:
+    yield from utils.dataflow_flex_template_run(
+        job_name=NAME,
+        template_path=flex_template_path,
+        bucket_name=bucket_name,
+        project=project,
+        region=location,
```

Comment:
Other dataflow examples default to the conftest.py region of us-central1. I think changing this to (presumably) us-west1 here will make this sample work, because there's availability in that region.

Attempting to change this value here to see if it will work (will revert if required)